aten/src/ATen/templates/TypeDefault.cpp

```diff
@@ -29,8 +29,7 @@ namespace TypeDefault {
 
 }  // namespace TypeDefault
 
-namespace {
-auto registerer = torch::import()
```

Comment:
I think you need update aten/src/ATen/templates/PerOpRegistration.cpp as well for internal buck build.